src/index.tsx

```diff
@@ -39,6 +39,7 @@ export {
   Header,
   Icon,
   Link,
+  LinkProps,
```

Comment:
I'm thinking this may not be necessary anymore since we were trying to fix that typescript error, but it turned out to be unreleated

Reply:
you are 100% correct. tested and removed.